### ISSUE TITLE
`fft(::Double64s)` ok on v1.3.0-DEV, not ok on v1.1.1 

### ISSUE THREAD

**Issue by JeffreySarnoff** [JuliaMath/AbstractFFTs.jl#28]
```
julia> VERSION # v"1.3.0-DEV.263"
julia> using DoubleFloats, FastTransforms, FFTW

julia> let df64s = rand(Double64, 2^10)
         ComplexDF32( sum(df64s .- ifft(fft(df64s))) )
       end
3.9988234e-30 + 0.0im
```

```
julia> VERSION # v"1.1.1"
julia> using DoubleFloats, FastTransforms, FFTW

julia> let df64s = rand(Double64, 2^10)
         ComplexDF32( sum(df64s .- ifft(fft(df64s))) )
       end
ERROR: type DoubleFloat{Float64} not supported
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] _fftfloat(::Type{DoubleFloat{Float64}}) at C:\Users\jas\.julia\packages\AbstractFFTs\PUqOK\src\definitions.jl:22
 [3] _fftfloat(::DoubleFloat{Float64}) at C:\Users\jas\.julia\packages\AbstractFFTs\PUqOK\src\definitions.jl:23
 [4] fftfloat(::DoubleFloat{Float64}) at C:\Users\jas\.julia\packages\AbstractFFTs\PUqOK\src\definitions.jl:18
 [5] complexfloat(::Array{DoubleFloat{Float64},1}) at C:\Users\jas\.julia\packages\AbstractFFTs\PUqOK\src\definitions.jl:31
 [6] fft(::Array{DoubleFloat{Float64},1}, ::UnitRange{Int64}) at C:\Users\jas\.julia\packages\AbstractFFTs\PUqOK\src\definitions.jl:198 (repeats 2 times)
 [7] top-level scope at REPL[3]:2
```

**Comment by daanhb** [comment]
Could it be that you are using release 0.4.2 of FastTransforms on julia 1.1, but the current master on version 1.3? This is the only way I can reproduce your error locally.

The "generic fft" part of FastTransforms overrides the `complexfloat` function in order to avoid throwing this error. It seems this has not made it into a release yet.

**Comment by giordano** [comment]
Yes, I noted a couple of days ago in https://github.com/JuliaMath/AbstractFFTs.jl/pull/29#issuecomment-496008870 that the feature is not yet in a released version of `FastTransforms.jl`

**Comment by daanhb** [comment]
Okay, thanks for confirming.

**Comment by JeffreySarnoff** [comment]
fixed in v0.5 of FastTransforms, thank you
